pkg/parser.go

```diff
@@ -273,6 +273,9 @@ func ParseValue(fieldName string, fieldType string, tz *time.Location, value int
 
 			return parseUInt64Value(value, isNullable)
 		case "Int64":
+			if fieldName == "t" {
```

Comment:
could you explain? do you use toInt64(dateTimeField) in your SQL query?

Current macros for DateTime handling usually use toUInt32

Reply:
The query is generated by the backend! I don't think I'm doing anything special? Could it be related to that I'm using datetime64?
Check out https://github.com/Vertamedia/clickhouse-grafana/issues/350 for more details.

Reply:
Yeah, the docs explicitly state DateTime64 is stored as Int64 https://clickhouse.tech/docs/en/sql-reference/data-types/datetime64/amp/